SachinYadav/NullPointerExeptionChallenge.java

```diff
@@ -0,0 +1,24 @@
+package Exercise2;
+
+/*
+ * This is debugging exercise-02 of project earlybird
+ * The corrected code after debugging and adding exception handling
+ * @author - Sachin Yadav
+ * @Organization - Veersa Technologies
+ */
+public class NullPointerExceptionChallenge {
+
+    public static void main(String[] args) {
+        // String str = "Sachin" it will work fine because the string is not null
+        // String str = null; // gives NullPointerExepception because str is empty 
+        String str = "Sachin";
```

Comment:
Instead of changing the String, add a check if the string is null then proceed to store the length and store it